### ISSUE TITLE
hab pkg binlink doesn't work in a studio

### ISSUE THREAD

**Issue by metadave** [habitat-sh/habitat#975]
This seems to happen to any package I try to binlink in a studio:

```
[3][default:/src:1]# hab pkg binlink core/redis redis-cli
» Symlinking redis-cli from core/redis into /bin
★ Binary redis-cli from core/redis/3.0.7/20160614001713 symlinked to /bin/redis-cli
[4][default:/src:0]# redis-cli
bash: redis-cli: command not found
[5][default:/src:127]# exit
logout

root@3f47518d40f6:/src# redis-cli
bash: redis-cli: command not found
root@3f47518d40f6:/src# hab pkg binlink core/redis redis-cli
» Symlinking redis-cli from core/redis into /bin
★ Binary redis-cli from core/redis/3.0.7/20160614001713 symlinked to /bin/redis-cli
root@3f47518d40f6:/src# redis-cli
Could not connect to Redis at 127.0.0.1:6379: Connection refused
```

**Comment by davidwrede** [comment]
Looks like you weren't in `/bin` when you tried to call redis-cli. Try that and see if it works. A good feature request would be to add `/bin` to PATH within the studio.

**Comment by metadave** [comment]
ah, the link is in `/bin`, but the PATH hasn't been updated. Would be a nice enhancement for the studio.

**Comment by fnichol** [comment]
Bingo, that's why! @metadave You could run `hab pkg binlink -d /hab/bin core/redis redis-cli` to get a result that would be on `PATH`, although that's a bit more work

**Comment by eeyun** [comment]
I think that the appropriate way to move forward on this (at least initially) is to document this behavior more fully. We have the functionality that the initial issue is asking for its just not clearly documented how you achieve said functionality.

We're going to go ahead and add this to our `Help Wanted` Milestone and get this behavior documented when we can!

**Comment by reset** [comment]
A good solution for this might be to make the default binlink destination directory configurable by an environment variable as well as with the flag. Right now, the default is always `/bin`. If we were to set `HAB_BINLINK_DIR=/hab/bin` inside the studio and reflect on that environment variable we'd probably get the experience we want and it should be a quick and easy fix!

**Comment by reset** [comment]
@eeyun this is still unresolved even though #2376 made things a hell of a lot better. We need to set the default binlink directory to `/hab/bin` instead of `/bin` since `/bin` is not on the path by default in the studio

**Comment by davidwrede** [comment]
Once #2915 gets merged, I'll update the docs.

**Comment by reset** [comment]
Closed by #2951
